canonical-web-and-design/ubuntu.com
# Update takeovers

## Done

- Removed the AI Edge Webinar
- Changed the CTA on the Dell Kubernetes takeover to 'Learn more'

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the aforementioned changes are in place.

## Issue / Card

Fixes #6682

## Screenshots

![image](https://user-images.githubusercontent.com/441217/74718093-76e3d180-5229-11ea-830f-63cb49ab82d5.png)
